src/core/icons/folderIcons.ts

```diff
@@ -1281,6 +1281,10 @@ export const folderIcons: FolderTheme[] = [
         folderNames: ['.cursor'],
         light: true,
       },
+      {
+        name: 'folder-metro',
+        folderNames: ['.metro'],
```

Comment:
The dot before the folder name is not needed. We add a dot automatically


```suggestion
        folderNames: ['metro'],
```